### PR TITLE
bilisimio android dersleri

### PR DESCRIPTION
**Pull request by cemdrman**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists  in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books-tr.md [hunk 1]
```diff
@@ -39,12 +39,12 @@
 * [Android Dersleri](https://umiitkose.com/android) - Ümit Köse
 * [Android Geleceği Yazanlar](https://gelecegiyazanlar.turkcell.com.tr/konu/android)
 * [Android Türkçe PDF](http://umiitkose.com/wp-content/uploads/2015/08/AndroidStudio.pdf) - Ümit Köse (PDF)
+* [Android Dersleri](https://bilisim.io/category/mobile/android) - Bilisimio
```
Comment (eshellman) [inline books/free-programming-books-tr.md:42]: Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

Comment (eshellman) [reply books/free-programming-books-tr.md:42]: But the resource doesn't seem to be a book.

**Review by eshellman — COMMENTED** [review]
Thanks for fixing the bad links. Not sure about the added resource though, can you explain why it belongs in a list of books?

File: books/free-programming-books-tr.md (hunk 1 above)
Comment (cemdrman) [reply books/free-programming-books-tr.md:42]: yes you are right but already there are some blog so I added too

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**